services/QuillLMS/client/app/bundles/Shared/components/shared/dataTableChip.tsx

```diff
@@ -19,6 +20,14 @@ export const DataTableChip = ({ color, icon, label, link }: DataTableChipProps)
       </a>
     )
   }
+  if (onClick) {
+    return (
+      <button className={`data-table-chip ${color} focus-on-light`} onClick={onClick}>
+        {icon && <img alt={icon.alt} src={icon.src} />}
+        <p>{label}</p>
```

Comment:
I think a `span` is the more appropriate semantic element here -- I've been getting a lot of console errors about nesting block elements inside inline elements, which I think is technically not allowed under the HTML5 specification. There are some cases where it's tricky to avoid (like nesting elements in tables), but I think we should try where we can.

Reply:
ok! I'll make that change. Thank you for the context